CHANGELOG.md

```diff
@@ -10,7 +495,7 @@ In this version, we've introduced features like window size API, enhanced chart
 The list of \*breaking changes\* can be found [here](https://github.com/ratatui-org/ratatui/blob/main/BREAKING-CHANGES.md) ⚠️.
 Also, we created various tutorials and walkthroughs in [Ratatui Book](https://github.com/ratatui-org/ratatui-book) which is available at <https://ratatui.rs> 🚀
 
-✨ **Release highlights**: <https://ratatui.rs/highlights/v0.24.html>
+✨ **Release highlights**: <https://ratatui.rs/highlights/v024>
```

Comment:
```suggestion
✨ **Release highlights**: <https://ratatui.rs/highlights/v025>
```

Reply:
This link is actually from the previous release - I just fixed it because it was 404

Reply:
Oh yeah my bad :+1:, looks good then!